README.md

```diff
@@ -1,100 +1,45 @@
-# COVID-19 Data Model
+# COVID-19 Data Pipeline
 
-*Objective:* Provide a library/API that ingests COVID-19 data to provide simulated outcomes based on local isolation/quarantine policy levers as represented in published models.
+COVID data pipeline / API supporting https://covidactnow.org/.
 
-This work supports https://covidactnow.org/.
-
-Check It Out via Jupyter:
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/covid-projections/covid-data-model/main)
-
-
-## Resources
-
-We track our work via Trello. Our [engineering board](https://trello.com/b/zLf79HJ8/public-engpmux) is public.
-
-Please file any issues via GitHub.
-
-We are always looking for more collaborators. See our [contact page](https://covidactnow.org/contact) for general information or reach out directly about helping via this [form](https://docs.google.com/forms/d/e/1FAIpQLSfQkdwXsbDbwLHhWwBD6wzNiw54_0P6A60r8hujP3qnaxxFkA/viewform).
-
-### Models
-
-* [R-Based Model](https://alhill.shinyapps.io/COVID19seir/)
-  * [Source Code](https://github.com/alsnhll/SEIR_COVID19)
-* [Penn-Chime](http://penn-chime.phl.io/)
-* [Imperial College Model](https://www.imperial.ac.uk/media/imperial-college/medicine/sph/ide/gida-fellowships/Imperial-College-COVID19-NPI-modelling-16-03-2020.pdf)
-* [Astor Sq ICU Beds Model](https://docs.google.com/spreadsheets/d/1DlC5kh9ve-Giv96XTnhCiB6vQAkQCjl5bDSjT68Q0FY/htmlview#)
-* [Epidemic Calculator built by Open AI member](https://gabgoh.github.io/COVID/index.html)
-* [Hopkins IDD Model](https://github.com/HopkinsIDD/COVIDScenarioPipeline)
-
-### Data Sources
-See [covid-data-public](https://github.com/covid-projections/covid-data-public) for data sources being used or considered.
-
-Some code in the `covid-data-model` repo depends on there being a copy of the `covid-data-public` repo at
-`../covid-data-public`.
+It ingests data scraped via https://github.com/covid-projections/can-scrapers, combines it, calculates metrics, and generates data files for the [Covid Act Now API](https://apidocs.covidactnow.org/) and [website](https://covidactnow.org/)
 
+# Development
 
 ## Setup
 
-Detailed setup instructions can be found [here](./SETUP.md)
-
-# API Snapshots
```

Comment:
I deleted the API Snapshots / Labeling snapshots sections since they're kind of out of date and you can kick these off via github UI now. So ideally we'd add instructions for using the github actions UI to kick off snapshots and maybe explain the daily build orchestration, but I didn't feel like doing that in this PR, so I'm just deleting this since it's not super helpful as-is.